src/Button.js

```diff
@@ -13,12 +13,21 @@ class Button extends Component {
   }
 
   componentDidMount() {
-    const { tooltip, tooltipOptions } = this.props;
-    if (
-      typeof $ !== 'undefined' &&
-      (typeof tooltip !== 'undefined' || typeof tooltipOptions !== 'undefined')
-    ) {
-      $(this._btnEl).tooltip(tooltipOptions);
+    if (!M) return;
+
+    const { tooltip, tooltipOptions = {}, fab } = this.props;
+    if (tooltip) {
+      this.instance = M.Tooltip.init(this._btnEl, tooltipOptions);
+    }
+
+    if (fab && this._floatingActionBtn) {
+      this.instance = M.FloatingActionButton.init(this._floatingActionBtn);
+    }
+  }
```

Comment:
I don't think you need to check for `this._floatingActionBtn` on L23

Reply:
No problem! You can checkout any file from any branch like so:
`git checkout master package-lock.json`

Reply:
Omg it worked :smile:

Now I will work on the others tasks:
- Remove check this._floatingActionBtn on L23
- Merge the docs of buttons and FloatingActionButtonsPage

:)

Reply:
Done :)
@alextrastero